service/prover/prover/prover.go

```diff
@@ -42,6 +42,16 @@ func WithRedis(redisType string, redisPass string) redis.Option {
 		p.Pass = redisPass
 	}
 }
+
+func IsBlockSizesSorted(blockSizes []int) bool {
+	for i := 1; i < len(blockSizes); i++ {
```

Comment:
`i` start from 1 may case index out of range.

Reply:
in that case, 1 should larger than `len(blockSizes)`